ntoskrnl/include/internal/mm.h

```diff
@@ -131,6 +131,13 @@ typedef ULONG_PTR SWAPENTRY;
 #define MM_ROUND_DOWN(x,s)                  \
     ((PVOID)(((ULONG_PTR)(x)) & ~((ULONG_PTR)(s)-1)))
 
+/* PAGE_ROUND_UP and PAGE_ROUND_DOWN equivalent for 64-bit-only data types */
```

Comment:
Naively to me it looks like they also work on 32-bit types as well (or, say, any type that supports these arithmetic/bitwise operations).
If this is so, you may adjust slightly the comment.